prac_06/myguitars.py

```diff
@@ -14,4 +14,28 @@ def main():
         cost = float(input("Cost: $"))
         guitar_to_add = Guitar(name, year, cost)
         guitars.append(guitar_to_add)
-        print(guitar_to_add,"added.")
\ No newline at end of file
+        print(guitar_to_add,"added.")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:  # lists, strings and other collections are False when empty, True when non-empty
+        # In order for sorting to work on Guitar objects,
+        # at least the __lt__ method must be defined in Guitar class
+        guitars.sort()
+        print("These are my guitars: ")
+        for i, guitar in enumerate(guitars):
+            vintage_string = ""
+            if guitar.is_vintage():
+                vintage_string = "(Vintage)"
+            print("Guitar {0}: {1.name:>30}({1.year}), worth ${1.cost:10,.2f}\
+             {2}".format(i + 1, guitar, vintage_string))
```

Comment:
The way this print statement is formatted is great. It is easy to read and you know exactly what variable you're putting into the place holder.